src/serac/physics/utilities/variational_form/detail/hexahedron_h1.inl

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.

```

Comment:
These inl files will not run through our code check logic.  Can you add ".inl" to https://github.com/LLNL/serac/blob/f1c4b6cf6a6afc0391fe227282e7bf4357a9aee5/cmake/SeracMacros.cmake#L35